eventlet/hubs/pyevent.py

```diff
@@ -1,12 +1,20 @@
 import sys
 import traceback
-import event
 import types
 
 from eventlet.support import greenlets as greenlet
 import six
 from eventlet.hubs.hub import BaseHub, READ, WRITE
```

Comment:
Would there be any value to making this (and any others) consistent with the changes above?

Reply:
Probably yes. I tried to make as few as possible changes to pyevent hub because it's not tested in CI, may be completely broken for all I know.

Reply:
Okay.